addons/base_vat/views/res_partner_views.xml

```diff
@@ -6,14 +6,18 @@
             <field name="name">view.partner.base.vat.form</field>
             <field name="inherit_id" ref="base.view_partner_form" />
             <field name="arch" type="xml">
-                <xpath expr="//field[@name='vat']" position="replace">
-                    <field name="perform_vies_validation" invisible="1"/>
+                <xpath expr="//div[hasclass('o_address_format')]" position="after">
```

Comment:
* Just to avoid diff (and possible position issues), I'd keep `perform_vies_validation` here
* can't we keep the same position `expr` here and only change the position? Or does that mess with the `move` below? I wouldn't want the fields to move place because of that new selector
```suggestion
                <xpath expr="//field[@name='vat']" position="after">
                    <field name="perform_vies_validation" invisible="1"/>
```

Reply:
seems to work fine without messing with the move